R/render.r

```diff
@@ -602,15 +582,17 @@ check_bootswatch_theme <- function(bootswatch_theme, bs_version, pkg) {
 
 pkgdown_bslib_defaults <- function() {
   list(
-    `navbar-nav-link-padding-x` = "1rem",
-    `primary` = "#0054AD",
-    `secondary` = "#e9ecef",
-    `navbar-bg` = "#f8f9fa",
-    `border-width` = "1px",
-    `code-bg` = "#f8f8f8",
-    `code-color` = "#333",
-    `fu-color` = "#4758AB",
-    `border-radius` = "1rem"
+    `navbar-nav-link-padding-x` = "1rem !default",
+    `primary` = "#0054AD !default",
+    `secondary` = "#e9ecef !default",
+    `navbar-bg` = "#f8f9fa !default",
+    `border-width` = "1px !default",
+    `code-bg` = "#f8f8f8 !default",
+    `code-color` = "#333 !default",
+    `fu-color` = "#4758AB !default",
+    `border-radius` = "1rem !default",
+    `navbar-light-color` = "rgba($fg, 0.8) !default",
+    `navbar-light-hover-color` = "rgba($fg, 0.9) !default"
```

Comment:
I can't use `$body-color` here as I get the error that it's not defined, but it's actually what I'd want to do.

Reply:
I think you probably want 

```r
`navbar-light-color` = "rgba(color-contrast($navbar-bg), 0.8) !default",
`navbar-light-hover-color` = "rgba(color-contrast($navbar-bg), 0.9) !default",
```